src/runtime_src/core/tools/common/XBHelpMenusCore.h

```diff
@@ -44,8 +44,9 @@ namespace XBUtilities {
                             const boost::program_options::options_description & _optionDescription,
                             const boost::program_options::options_description &_optionHidden,
                             const boost::program_options::positional_options_description & _positionalDescription,
-                            const boost::program_options::options_description &_globalOptions,
-                            bool removeLongOptDashes = false);
+                            const boost::program_options::options_description &_globalOptions,
+                            bool removeLongOptDashes = false,
+                            const std::string customHelpSection = "");

```

Comment:
```suggestion
                            const std::string& customHelpSection = "");

```
This should compile just fine as we are binding the temporary value to the reference! Pretty neat